storage/feed.go

```diff
@@ -79,6 +79,33 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	Desc *proto.StoreDescriptor
```

Comment:
I opted to simply use it in this commit: i'm now recording capacity statistics as time series.  This is actually something helpful that I should have done anyway.